### PR TITLE
Minor: Docs Enhancement (WIP)

### LINKED ISSUE #1280: replace "the Helium app" with specific app
inconsistent usage, "the Helium Hotspot app" or "the Helium Wallet app" to replace these

### LINKED ISSUE #1700: Consider completing the update to Ledger Hardware Wallet User
## Context

Screenshot 2023-12-22 at 9 46 55 AM

## Problem

What about this page isn't updated? If it isn't updated, please remove this notice.

### LINKED ISSUE #1703: Consider adding verification step at the end of migration tool
## Context

https://docs.helium.com/solana/migration/ledger/#migration-tool

## Problem

After the last step, there is no verification step. It would be helpful if the docs described a step so that one can manually verify that the migration completed successfully. 

## Proposal

I imagine that step would look like:

1. Find the public address associated with your pre-migrated account. Note: how do I do this?
2. Search for your old public address on https://explorer.solana.com/ and ensure the account has no funds
3. Find the public address associated with your post-migrated account. 
4. Search for your new public address on https://explorer.solana.com/ and ensure the account has some funds

Note: 
- I'm observing some residual [Helium Network Token](https://explorer.solana.com/address/hntyVP6YFm1Hg25TN9WGLqM12b8TQmcknKrdu1oxWux) in the pre-migrated account. If this is expected behavior, I think it should be documented here.
- It would nice to add a note that says something like: "staked HNT will be migrated to a single [veHNT token](https://explorer.solana.com/address/HBgj24hUAAQs5ghzrrrM7T4WgZ7CQm3JiSUm3sUhTwGb)" with a link to docs on the veHNT token.

### LINKED ISSUE #1830: Fix Broken Link to Installation Guide
Create Contribution Guide: Local Server Setup for Documentation

There is currently no contribution guide available for setting up a local server to contribute to the project's documentation.

The existing link to the installation guide is broken, leaving new contributors without clear instructions. To address this,  a straightforward "Contribution Guide: Local Server Setup" in the CONTRIBUTING.md file should be created

This guide will provide step-by-step instructions on installing Node.js, Yarn, and running the local server to view and contribute to the documentation.

i think it would make it easier for new or more  contributors to get started!

### PR DESCRIPTION
**Pull request by nermalcat69**
work in progress

issues
/closes #1830
/closes #1280
/closes #1700
/closes #1703

**Changes**
- Changed about-helium to getting-started
- enhanced the getting-started page
- Made changes in contributing.md
- Added verification step at the end of migration tool (#1703)

### CODE REVIEW

**Comment by nermalcat69** [comment]
add the solscribe label whenever you get time 👍🏼

File: package-lock.json
Comment (jthiller) [inline package-lock.json:1]: This diff probably shouldn't be here or be this drastic. Can you make sure it matches `master`?
Same with `yarn.lock`

Comment (nermalcat69) [reply package-lock.json:1]: sureee, im not sure why that happened but yeah

**Comment by nermalcat69** [comment]
matched the yark lock file, i hope its alright now

**Comment by nermalcat69** [comment]
fixed it

**Comment by jthiller** [comment]
Thank you for submitting updates to the Helium Docs as part of the Solana Scribes Hackathon! Please follow the instructions in this form so we may begin issuing awards.

https://forms.gle/z4ovMHYDo7pzy8TN6  

Please complete this form by March 15, 2024. (Approximately 1 week from today). We had SO MANY great submissions and we’re eager to distribute the awards to all of the contributors.

**Comment by jthiller** [comment]
Apologies, something happened to the Google forms link. This one should work: [https://forms.gle/z4ovMHYDo7pzy8TN6](https://forms.gle/z4ovMHYDo7pzy8TN6)

**Comment by nermalcat69** [comment]
nermalgato

**Pull request closed without merging**